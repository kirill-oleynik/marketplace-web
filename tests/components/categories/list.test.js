const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const CategoriesList = require('../../../app/components/categories/list').default;

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <CategoriesList
        t={(translation) => translation}
        expand={() => {}}
        collapse={() => {}}
        categories={[
          { id: 1, title: 'Test1', applications: [] },
          { id: 2, title: 'Test2', applications: [
            'app_one',
            'app_two'
          ] }
        ]}
      />
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
