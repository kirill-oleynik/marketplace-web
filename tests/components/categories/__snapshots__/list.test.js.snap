// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<div>
  <div>
    <Translate(Category)
      category={
        Object {
          "applications": Array [
            "app_one",
            "app_two",
          ],
          "id": 2,
          "title": "Test2",
        }
      }
      collapse={[Function]}
      expand={[Function]}
      verticalLayout={false}
    />
  </div>
</div>
`;
